modules/phase_field/src/kernels/CoupledMaterialDerivative.C

```diff
@@ -0,0 +1,68 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "CoupledMaterialDerivative.h"
+
+template <>
+InputParameters
+validParams<CoupledMaterialDerivative>()
+{
+  InputParameters params = validParams<Kernel>();
+  params.addRequiredCoupledVar("v", "Variable for the parital derivatives of the free energy");
+  params.addClassDescription("Kernel that implements (dFdv, test)");
```

Comment:
Minor: Could you put this particular parameter either at the top or the bottom of the list? It seems out of place in the middle.